src/inc/readytorun.h

```diff
@@ -186,6 +186,7 @@ enum ReadyToRunFixupKind
     READYTORUN_FIXUP_DeclaringTypeHandle        = 0x2D,
 
     READYTORUN_FIXUP_IndirectPInvokeTarget      = 0x2E, /* Target of an inlined pinvoke */
+    READYTORUN_FIXUP_IndirectPInvokeTargetSuppressGC  = 0x2F, /* Target of an inlined pinvoke (suppress GC) */
```

Comment:
Could you add this value to r2rdump (src\tools\r2rdump\R2RConstants.cs and handling in R2RSignature.cs)?

Reply:
Done.